tests-manual/C/services/core/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.6)
```

Comment:
Not a comment on this file, but rather a request to add a readme that explains what to do to run this manual test.